lms/templates/application_instances/new_application_instance.html.jinja2

```diff
@@ -7,19 +9,67 @@
      return url;
   }
 
-  function handleSubmit(form) {
+  function handleSubmit(event, form) {
+    if(!validateDomain(form.elements['lms_url']) || !validateEmail(form.elements['email'])) {
+      event.preventDefault();
+      return false;
+    }
     form.elements['lms_url'].value = addHttp(form.elements['lms_url'].value);
   }
+
+  function validateEmail(input) {
+    var parent = input.parentElement;
+    if (input.value.indexOf('@') === -1) {
+      parent.classList.add('has-error');
+      parent.getElementsByClassName('error')[0].innerHTML = 'Please enter a valid email address';
+      return false;
+    }
+    parent.classList.remove('has-error');
+    parent.getElementsByClassName('error')[0].innerHTML = '';
+    return true;
+  }
+
+  function validateDomain(input) {
+    var parent = input.parentElement;
+    if (
+        input.value.indexOf('.com') === -1 &&
```

Comment:
This seems a little bit fragile. I wonder if we could change it to be a regex that checks for `<domain name>.<top level domain name>`? Or if we were less concerned even just checking that it is non-empty.

Reply:
I would say we are probably less concerned.